python/tests/io/test_pyarrow.py

```diff
@@ -1130,3 +1131,15 @@ def test_projection_filter_on_unknown_field(schema_int_str: Schema, file_int_str
         _ = project(schema, [file_int_str], GreaterThan("unknown_field", "1"), schema_int_str)
 
     assert "Could not find field with name unknown_field, case_sensitive=True" in str(exc_info.value)
+
+
+def test_pyarrow_to_schema_simple(table_schema_simple: Schema, pyarrow_schema_simple: pa.Schema) -> None:
```

Comment:
Can we move tests to their own test file? We want to avoid the test files getting so large.

Reply:
Sure! I separated these tests into 4 files: `test_pyarrow`, `test_pyarrow_expression`, `test_pyarrow_projection`, and `test_pyarrow_projection`